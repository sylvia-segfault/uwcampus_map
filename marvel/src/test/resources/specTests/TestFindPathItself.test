# test the edge case when it tries to find a path from
# a character in the dataset to itself

LoadGraph graph biggerData.tsv
FindPath graph B B