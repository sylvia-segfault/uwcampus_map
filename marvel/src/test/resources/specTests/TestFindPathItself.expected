# test the edge case when it tries to find a path from
# a character in the dataset to itself

loaded graph graph
path from B to B: